libqtile/backend/wayland/core.py

```diff
@@ -155,6 +156,8 @@ def __init__(self):
         XdgOutputManagerV1(self.display, self.output_layout)
         ScreencopyManagerV1(self.display)
         GammaControlManagerV1(self.display)
+        self._output_power_manager = OutputPowerManagerV1(self.display)
+        self.add_listener(self._output_power_manager.set_mode_event, self._on_output_power_manager_set_mode)
```

Comment:
It looks like we don't need to keep around a handle to the manager as an attribute here.
```suggestion
        output_power_manager = OutputPowerManagerV1(self.display)
        self.add_listener(output_power_manager.set_mode_event, self._on_output_power_manager_set_mode)
```